graphql2/graphqlapp/alert.go

```diff
@@ -578,3 +589,27 @@ func (m *Mutation) UpdateAlertsByService(ctx context.Context, args graphql2.Upda
 
 	return true, nil
 }
+
+func (a *Alert) Meta(ctx context.Context, alert *alert.Alert) ([]graphql2.AlertMetadata, error) {
+	md, err := a.AlertStore.Metadata(ctx, alert.ID)
+	if err != nil {
+		return nil, err
+	}
+	var alertMeta []graphql2.AlertMetadata
+	for k, v := range md {
+		alertMeta = append(alertMeta, graphql2.AlertMetadata{Key: k, Value: v})
+	}
+	return alertMeta, nil
+}
+
+func (a *Alert) MetaValue(ctx context.Context, alert *alert.Alert, key string) (*string, error) {
+	md, err := a.AlertStore.Metadata(ctx, alert.ID)
+	val := ""
+	if err != nil {
+		return &val, err
+	}
+	if value, ok := md[key]; ok {
+		val = value
+	}
+	return &val, nil
```

Comment:
nit: let's simplify this, also if we get a not-found error, return empty string
```suggestion
	md, err := a.AlertStore.Metadata(ctx, alert.ID)
	if errors.Is(err, sql.ErrNoRows) {
		return new(string), nil // note: if we change the schema to `String!` we can clean this up
	}
	if err != nil {
		return nil, err
	}
	return &md[key], nil
```